src/log-uploader.ts

```diff
@@ -25,11 +25,13 @@ export interface IQueueItem {
 
 const logUploaderMap = new Map<string, LogUploader>();
 
-export const getOrCreateLogUploader = (url: string, debugMode = false): LogUploader => {
+export const getOrCreateLogUploader = (url: string, debugMode = false, useUniqueLogUploader?: boolean): LogUploader => {
   let uploader = logUploaderMap.get(url);
```

Comment:
Do you think it would make any sense to only do this `get` if we're not using a unique log uploader?